cmd/frcli/node_audit.go

```diff
@@ -41,6 +71,12 @@ var onChainReportCommand = cli.Command{
 			Name:  "enable_fiat",
 			Usage: "Create a report with fiat conversions.",
 		},
+		cli.StringSliceFlag{
```

Comment:
What do you think of using JSON instead? We do that in `lnd` in some places.
You could then just specify:
```
--categories='{"name":["regex1","regex2"],"name2":["regex"]}'
```
Or if you want to allow overwriting the boolean flags as well (assuming they're true by default):
```
--categories='{"name":{"patterns":["regex1","regex2"],"offChain":false},"name2":{"patterns": ["regex"]}}'
``

Reply:
Yeah this is definitely better! Updated to allow json input, decided to just go with the rpc struct directly since it's already annotated.

Reply:
Nice!